projects.js

```diff
@@ -0,0 +1,113 @@
+const projects = [
+  {
+    title: "Multi-Post Stories",
+    company: "CANPOY",
+    position: "Back End Dev",
+    date: "2015",
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    languages: ["html", "css", "javascript"],
+    imageUrl: "images/project1.png",
+    id: "1",
+  },
+  {
+    title: "Multi-Post Stories",
+    company: "Tonic",
+    position: "Back End Dev",
+    date: "2015",
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,",
+    languages: ["html", "css", "javascript"],
+    imageUrl: "images/project2.png",
+    id: "2",
+  },
+];
```

Comment:
😀GOOD JOB GUYS 🙌😀
- [ ] Please add 2 more projects to make it perfect, ok 👍? _The more projects you have, the easier it is to know you and your experience well._ Will all these steps your project should be completed 🍀